### ISSUE TITLE
Signup/Login Without Google

### ISSUE THREAD

**Issue by MelissaUpshaw** [charleswritescode/moodboard-webapp#3]
Hi, thanks for your wonderful web app. 👍
Does Firebase support signups/logins using other methods than Google?

**Comment by charleswritescode** [comment]
Yes Firebase supports Email, Twitter, Facebook, Google and Custom authentication.
I am planning on adding Email Login to this app soon
